adapters/nextmillennium/nextmillennium.go

```diff
@@ -127,7 +111,11 @@ func createBidRequest(prebidBidRequest *openrtb2.BidRequest, params *openrtb_ext
 		placementID = fmt.Sprintf("g%s;%s;%s", params.GroupID, size, domain)
 	}
 
-	bidRequest.Ext.Prebid.StoredRequest.ID = placementID
+	jsonExt := json.RawMessage(`{"prebid":{"storedrequest":{"id":"` + placementID + `"}}}`)
+
+	bidRequest := *prebidBidRequest
+	bidRequest.Ext = jsonExt
+	bidRequest.Imp[0].Ext = jsonExt
```

Comment:
Just want to double check this, are we sure that these two lines wouldn't overwrite potential information that already exists at `bidrequest.Ext` or `bidRequest.Imp[0].ext`?

Reply:
Thanks!
I changed it